### ISSUE TITLE
jumplinux1 ssh fails with 'Permission denied (publickey)' error

### ISSUE THREAD

**Issue by doherty100** [Azure-Samples/azuresandbox#12]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ X ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Install azuresandbx release v2.8.5
> Complete smoke testing in terraform-azruerrm-vnet-shared, ssh to jumplinux1 fails with error `Permission denied (publickey)`

### Any log messages given by the failure
> N/A

### Expected/desired behavior
> SSH to jumplinux1 should succeed

### OS and Version?
> jumplinux1: Ubuntu 22.04 LTS

### Versions
> v2.8.5

### Mention any other details that might be useful

> This is a breaking change, tested v2.8.4 to rule out regression, issue reproduced on that build.
> Thanks! We'll be in touch soon.

**Comment by doherty100** [comment]
This is a breaking change in Ubuntu 22.04.3 LTS release for Azure. The format of /etc/ssh/sshd_config has changed. Fix is on the way.

**Comment by doherty100** [comment]
Fixed in release [v2.8.6](https://github.com/Azure-Samples/azuresandbox/releases/tag/v2.8.6).
